src/include/sof/common.h

```diff
@@ -28,6 +28,8 @@
 	({const typeof(((type *)0)->member)*__memberptr = (ptr); \
 	(type *)((char *)__memberptr - offsetof(type, member)); })
 
+#define __section(x) __attribute__((section(x)))
+
```

Comment:
In another PR ?

Reply:
Yes, #2450

Reply:
ok, so this wont be in final version ?

Reply:
Mentioned PR has been merged, so I drop this commit